users/postgres/init.go

```diff
@@ -54,11 +54,17 @@ func migrateDB(db *sqlx.DB) error {
 				Up: []string{
 					`CREATE TABLE IF NOT EXISTS users (
 						email	 VARCHAR(254) PRIMARY KEY,
-						password CHAR(60)	  NOT NULL
+						password CHAR(60)	  NOT NULL,
```

Comment:
Adding this field here is a breaking change for the existing users. As the table is already there the migration will not be executed. This field addition should be added in the new "users_2"  migration as well.